stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2023 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.stock_barcodes.tests.test_stock_barcodes_picking import (
+    TestStockBarcodesPicking,
+)
+
+
+@tagged("post_install", "-at_install")
+class TestStockBarcodesPickingBatch(TestStockBarcodesPicking):
+    def setUp(self):
```

Comment:
Prefer classmethod instead.

Reply:
Upps!!! +1

Reply:
And please include the context keys for speeding up tests.

Reply:
Done!